### PR TITLE
DO-177: UX issue fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 1]
```diff
@@ -123,8 +125,8 @@ class MemberDetails extends React.Component {
                       <div className="details__col-50">
                         <div className="mbr-details__main-section">
                           <div className="name">
-                            <h2>{`${personalDetails.salutation} ${personalDetails.firstName} ${personalDetails.middleName} ${personalDetails.lastName}`}</h2>
-                            <p>{`${professionalDetails.profession}`}</p>
+                            <h2>{personalDetails.salutation ? personalDetails.salutation : ''} {personalDetails.firstName ? personalDetails.firstName : ''} {personalDetails.lastName ? personalDetails.lastName : ''}</h2>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:128]:
```suggestion
                            <h2>{personalDetails.salutation || ''} {personalDetails.firstName || ''} {personalDetails.lastName || ''}</h2>
```

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 2]
```diff
@@ -123,8 +125,8 @@ class MemberDetails extends React.Component {
...
+                            <p>{professionalDetails.profession ? professionalDetails.profession : ''}</p>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:129]:
```suggestion
                            <p>{professionalDetails.profession || ''}</p>
```

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 3]
```diff
@@ -153,69 +155,69 @@ class MemberDetails extends React.Component {
                               {socialNetworks.facebook ?
                                 <li className="facebook">
                                   <a href={socialNetworks.facebook} target="_blank" rel="noopener">
-                                    <IconFacebook />
+                                    <img src={SOCIAL_LOGO_FACEBOOK} alt="Facebook" />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:158]: Use Image component

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 4]
```diff
@@ -153,69 +155,69 @@ class MemberDetails extends React.Component {
...
                                   </a>
                                 </li>
                                 :
                                 <li>
                                   <a className="disable" title="Not Available" target="_blank" rel="noopener">
-                                    <IconFacebook />
+                                    <img src={SOCIAL_LOGO_FACEBOOK} alt="Facebook" />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:164]: Here as well

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 5]
```diff
@@ -153,69 +155,69 @@ class MemberDetails extends React.Component {
...
                                   <a href={socialNetworks.youtube} target="_blank">
-                                    <IconYouTube />
+                                    <img src={SOCIAL_LOGO_YOUTUBE} alt="YouTube" />
                                   </a>
                                 </li>
                                 :
                                 <li>
                                   <a className="disable" title="Not Available" target="_blank">
-                                    <IconYouTube />
+                                    <img src={SOCIAL_LOGO_YOUTUBE} alt="YouTube" />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:220]: Replace all  with Image component

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx [hunk 6]
```diff
@@ -141,10 +142,15 @@ class MemberProfileView extends React.Component {
...
+                          <div className="edit-icon__container">
+                            <Link to={Routes.MEMBER_PROFILE_EDIT} title="Edit Profile">
+                              <IconEdit />
+                            </Link>
+                          </div>
                           <div className="mbr-details__main-section">
                             <div className="name">
-                              <h2>{`${personalDetails.salutation} ${personalDetails.firstName} ${personalDetails.middleName} ${personalDetails.lastName}`}</h2>
-                              <p>{`${professionalDetails.profession}`}</p>
+                              <h2>{personalDetails.salutation ? personalDetails.salutation : ''} {personalDetails.firstName ? personalDetails.firstName : ''} {personalDetails.lastName ? personalDetails.lastName : ''}</h2>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx:152]:
```suggestion
                              <h2>{personalDetails.salutation || ''} {personalDetails.firstName || ''} {personalDetails.lastName || ''}</h2>
```

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx [hunk 7]
```diff
@@ -173,69 +179,69 @@ class MemberProfileView extends React.Component {
                                 {socialNetworks.facebook ?
                                   <li className="facebook">
                                     <a href={socialNetworks.facebook} target="_blank" rel="noopener">
-                                      <IconFacebook />
+                                      <img src={SOCIAL_LOGO_FACEBOOK} alt="Facebook" />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx:182]: Replace all the img tags with Image component

File: src/utils/common/formatters/formmatters.js [hunk 8]
```diff
@@ -44,6 +44,11 @@ export const removeLastString = (item) => {
   if (personalDetails.charAt((personalDetails.length) - 1) === ',') {
     personalDetails = personalDetails.substring(0, (personalDetails.length) - 1);
   }
+  if (personalDetails === '') {
```
Comment (mohitsaladiya) [inline src/utils/common/formatters/formmatters.js:47]:
```suggestion
  if (!personalDetails) {
    return '';
  }
```

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 1 above)
Comment (ayushmalik2110) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:128]: And use de-structure for all the fields of personal details.

File: src/utils/common/formatters/formmatters.js (hunk 8 above)
Comment (ayushmalik2110) [reply src/utils/common/formatters/formmatters.js:47]: Or you can use personalDetails.length as well.

Comment (ayushmalik2110) [reply src/utils/common/formatters/formmatters.js:47]: Use object de-structuring.

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx (hunk 6 above)
Comment (ayushmalik2110) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx:152]: Use object de-structuring.

File: src/utils/common/formatters/formmatters.js (hunk 8 above)
Comment (amitjain24524) [reply src/utils/common/formatters/formmatters.js]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx (hunk 7 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx (hunk 6 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 5 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: DONE

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: Done

**Pull request merged**